files/dll.js/info.ini

```diff
@@ -0,0 +1,5 @@
+author = "dnp_theme"
+github = "https://github.com/thednp/dll.js"
+homepage = "http://themeforest.net/user/dnp_theme"
```

Comment:
Should be relevant to the library being uploaded
On Apr 15, 2015 2:20 PM, "thednp" <notifications@github.com> wrote:

> In files/dll.js/info.ini
> <https://github.com/jsdelivr/jsdelivr/pull/4330#discussion_r28447381>:
>
> > @@ -0,0 +1,5 @@
> > +author = "dnp_theme"
> > +github = "https://github.com/thednp/dll.js"
> > +homepage = "http://themeforest.net/user/dnp_theme"
>
> Well, it's relevant to the author, should it be to the file?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jsdelivr/jsdelivr/pull/4330/files#r28447381>.
>